src/xtgeo/common/sys.py

```diff
@@ -421,7 +421,7 @@ def get_cfhandle(self) -> int:
 
         if _cfhandle already exists, then _cfhandlecount is increased with 1
 
-        Returns:
+        Returns
             int indicating the file handle number.
```

Comment:
I'm a bit confused -- the colon is removed after the `Returns` in some cases here but preserved in many others. This seems inconsistent on its own, but maybe there's a reason I don't understand. Is this caused by a particular code?